tools/deployment/pytorch2onnx.py

```diff
@@ -280,6 +285,10 @@ def parse_args():
         help='Whether to export model without post process. Experimental '
         'option. We do not guarantee the correctness of the exported '
         'model.')
+    parser.add_argument(
+        '--normalize-in-graph',
+        action='store_true',
+        help='Whether to include image normalization in ONNX graph.')
```

Comment:
Could update the doc as well:
https://github.com/open-mmlab/mmdetection/blob/master/docs/tutorials/pytorch2onnx.md#usage

Reply:
If a user wants to deploy a model on TensorRT by PyTorch->ONNX->TensorRT, then, this optional feature would have an influence on `onnx2tensorrt.py` tool.

Reply:
I've updated the docs and added a `--skip-normalize` option to the `onnx2tensorrt.py` tool. However, I do not have MMCV built with the tensorRT support, so I am not able to test the change myself.

Reply:
You could follow the instructions in this doc [onnx2tensorrt](https://github.com/open-mmlab/mmdetection/blob/master/docs/tutorials/onnx2tensorrt.md) to test the logic.

Reply:
I'm developing on MacOS, so I can't install TensorRT. Regardless, I've tested that the option is properly picked up by the argparse and that the logic is correct in not including the `normalize_cfg` entry in `input_config` when the `--skip-normalize` is set to `True`.